code/__DEFINES/clothing.dm

```diff
@@ -49,6 +37,49 @@
 
 #define slots_amt			20 // Keep this up to date!
 
+//I hate that this has to exist

```

Comment:
One is where it is, and one is where it fits? And the two have no relation  to each other?